### ISSUE TITLE
Small Bug Adding Forces to Channels

### ISSUE THREAD

**Issue by ShendMikullovci** [serge-web/serge#1414]
Selecting a force to add to a channel from the dropdown list bugs the addition:

https://user-images.githubusercontent.com/85491794/158082679-95534fc5-5a1d-4fdd-bae0-3387e2f0c358.mp4

The dropdown to restrict/allow access to certain roles is also dysfunctional: 

![Roles Dropdown Channels Dysfunctional](https://user-images.githubusercontent.com/85491794/158082796-7c5c6ea6-c9d4-42ad-9d35-2c322334f77d.png)

**Comment by ShendMikullovci** [comment]
> Selecting a force to add to a channel from the dropdown list bugs the addition:
> 
>  Adding.Forces.to.Channels.Bug.mp4 

It appears this bug is only present with Mapping channels so far.

**Comment by IanMayo** [comment]
Ok, we may need to improve how we define participation in `Mapping` channels. 
* I'm pretty sure we should have a checkbox for `can submit orders` - it seems better to apply it here, than in the `roles` data. We should have checking to make sure that a row with `can submit orders` only has one name role in it, and that we only allow one named role per force to be able to submit orders.
* I'm also pretty sure we don't need the `Actions` drop-down.  The above checkbox is enough to control the level of participation.

**Comment by ShendMikullovci** [comment]
Is this how you envisioned it? : 

![New Mapping Participation](https://user-images.githubusercontent.com/85491794/158178327-d9c01387-cb94-4983-add1-667d2dfa9cef.png)

If so, how do we specify what kind of orders do we want each player to be able to give? New column?

**Comment by IanMayo** [comment]
Yes, a checkbox column.  But, we can't allow `all roles` in a force to submit orders, unless there is only one force for the role.

We don't specify which kind of orders to give.  The player plans their turn using the mapping controls we provide them, then they click on `Submit orders`.    So, there isn't a specific template, Serge itself collates the data into the message.

**Comment by ShendMikullovci** [comment]
> We don't specify which kind of orders to give. 

The reason I ask is that our current channel includes these actions: 

![Current Actions Mapping Channel](https://user-images.githubusercontent.com/85491794/158195415-79562ed0-3cef-4337-b5f2-17b6ef2a8531.png)

I was thinking how we could incorporate these actions with that new design.

**Comment by IanMayo** [comment]
Sure, for a mapping channel we don't specify a template.

That means - the columns for `participants` changes depending on what type of channel it is.  In a similar way, a `Chat Channel` doesn't have a templates drop-down, since they are all chat messages.
